pennylane/templates/subroutines/phase_adder.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
I think this is the correct copyright line.
```suggestion
# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Reply:
The original was correct.